tools/bin/ci_integration_test.sh

```diff
@@ -13,7 +13,17 @@ if [[ "$connector" == "all" ]] ; then
   echo "Running: ./gradlew --no-daemon --scan integrationTest"
   ./gradlew --no-daemon --scan integrationTest
 else
-  if [[ "$connector" == *"connectors"* ]] || [[ "$connector" == *"bases"* ]]; then
+  if [[ "$connector" == *"base-normalization"* ]]; then
+    selected_integration_test="base-normalization"
+    integrationTestCommand="$(_to_gradle_path "airbyte-integrations/bases/base-normalization" integrationTest)"
+    export SUB_BUILD="NORMALIZATION"
+    ./gradlew --no-daemon --scan airbyteDocker
```

Comment:
I don't quite understand why `:airbyte-integrations:bases:base-normalization:airbyteDocker` is not executed when calling `:airbyte-integrations:bases:base-normalization:integrationTest`

even though `customIntegrationTestPython.dependsOn ':airbyte-integrations:bases:base-normalization:airbyteDocker'` ...

So to make the tests passed in the CI, I couldn't figure out anything other than explicitly invoking it here... maybe there's a better way?

Reply:
yeah I remember looking at this with you and being surprised too.

this seems 'ok' as a short term fix. we should definitely create a follow up issue and look into why this is happening when we have time (after Beta launch).